scripts/installs/start_glassfish_service.bat

```diff
@@ -1 +1,3 @@
-net start "domain1 GlassFish Server"
\ No newline at end of file
+Taskkill /IM domain1Service.exe /F
+ping 192.168.200.201 -n 1 -w 1000 > nul
```

Comment:
In my (admittedly limited) research, pinging an unavailable address is one of the only reliable ways to introduce a delay (like "sleep") into a windows batch script. Happy to replace this with something better if anyone has recommendations.